.github/workflows/release-log.yml

```diff
@@ -0,0 +1,22 @@
+name: Log Release PR
+
+on:
+  pull_request:
+    branches:
+      - release
```

Comment:
Should we add the v8 release branch to capture v8 releases?

Reply:
Added.